.github/workflows/flake8.yml

```diff
@@ -1,17 +1,17 @@
 name: Flake8
 on: pull_request
 job:
```

Comment:
should be `jobs:`

Reply:
so, if I merge and do another change on bad.py how am I should see the error?

Reply:
The CI should fall but with other error regarding the `bad.py` and not about the `flake8.yaml` syntax as it was.